pkg/controller/validatingadmissionpolicystatus/controller.go

```diff
@@ -156,6 +155,6 @@ func (c *Controller) reconcile(ctx context.Context, policy *v1.ValidatingAdmissi
 			WithObservedGeneration(policy.Generation).
 			WithTypeChecking(admissionregistrationv1apply.TypeChecking().
 				WithExpressionWarnings(warningsConfig...)))
-	_, err := c.policyClient.ApplyStatus(ctx, applyConfig, metav1.ApplyOptions{FieldManager: ControllerName, Force: true})
+	_, err := c.policyClient.ApplyStatus(ctx, applyConfig, metav1.ApplyOptions{FieldManager: c.name, Force: true})
```

Comment:
ControllerName is `"validatingadmissionpolicy-status"` but the constant value is `"validatingadmissionpolicy-status-controller"`.

should I hardcode it with the previous value from ControllerName?

Reply:
/cc @jiahuif

Reply:
@atiratree How to deal with this problem? Can you give me some hints? Thanks a lot!